### ISSUE TITLE
Excessive windows crashing

### ISSUE THREAD

**Issue by JohnnyMakeBeliever** [kolton/d2bot-with-kolbot#1011]
Any solution to this? It happens mainly during chestmania and in act 5, I'm only running it on -w parameters and starting hidden + unchecked visibility as mentionned in the FAQ but to no success in fixing

Windows 8 (again mentionned as happenning alot in the FAQ) running Win 7 compatibility with all the framework required

**Comment by semirotta** [comment]
Seems like it wont crash as often if -ns is off from the parameters, I have also noticed that act4 and 5 cause a lot of crashing. Its not alone the chestmania though. Just said acts it seems.

**Comment by JohnnyMakeBeliever** [comment]
That's the thing the only parameter I'm running is -w, and for me act 4 doesn't seem to crash it's mostly during act 5 or chestmania

**Comment by semirotta** [comment]
Right, well I cant help. Here lurking and waiting for someone who has more knowledge aswell ^^

**Comment by Spink2019** [comment]
Tons of windows crashing...I upgraded to newest version. Reinstall. Manual edited all config again. Only copied my pickit over. Windows 10 Admin Mode. Back to the older version I downloaded a month ago.

**Comment by Spink2019** [comment]
Well...back to crashing alot...weird. Was good for 40 runs.

**Comment by Spink2019** [comment]
Specifically my follower Sorc in Throne.

**Comment by nineinchnailzpsn** [comment]
Same here.  Crashes in chest mania almost every time randomly.

**Comment by iihieiii** [comment]
starting to get the same here =/

**Comment by skim063** [comment]
Found a temp solution.
1. Go to char config and under General config change min and maxgametime. This is so your bot doesnt crash log off after a certain time gap. I found out my MaxGametime was =600 which is roughly 10 mins max games. 

	// General config
	Config.AutoMap = false; // Set to true to open automap at the beginning of the game.
	Config.LastMessage = ""; // Message or array of messages to say at the end of the run. Use $nextgame to say next game - "Next game: $nextgame" (works with lead entry point)
	Config.MinGameTime = 60; // Min game time in seconds. Bot will TP to town and stay in game if the run is completed before.
	Config.MaxGameTime = 0; // Maximum game time in seconds. Quit game when limit is reached.
	Config.TeleSwitch = false; // Switch to secondary (non-primary) slot when teleporting more than 5 nodes.
	Config.OpenChests = false; // Open chests. Controls key buying.
	Config.MiniShopBot = true; // Scan items in NPC shops.
	Config.PacketShopping = false; // Use packets to shop. Improves shopping speed.
	Config.TownCheck = false; // Go to town if out of potions
	Config.LogExperience = false; // Print experience statistics in the manager.
	Config.PingQuit = [{Ping: 0, Duration: 0}]; // Quit if ping is over the given value for over the given time period in seconds.

2. Secondly. Set D2BS to run under windows 7 or service pack 3.
See link below and follow steps.

https://darcvigilante.site/showthread.php?tid=155


So far after 15 straight runs around 12 mins no crash at all.

**Comment by semirotta** [comment]
Ofc you should be looking at your minimal and maximum game times.
10 minutes per game is not enough and easily gets you shadow banned.
